internal/component/azure/servicebus/subscription.go

```diff
@@ -175,7 +176,11 @@ func (s *Subscription) ReceiveAndBlock(handler HandlerFunc, lockRenewalInSec int
 		s.logger.Debugf("Received message: %s; current active message usage: %d/%d", msg.MessageID, len(s.activeMessagesChan), cap(s.activeMessagesChan))
```

Comment:
Perhaps the log here could include msg.Sequencenumber too? So there's some correlation in the logs

Reply:
I've added the sequence number to the debug log when adding and removing the active message as we'll have checked for the existence of the sequence number by then and it's contextual. Happy with that?